plugins/Sitemaps/README.md

```diff
@@ -0,0 +1,18 @@
+# Sitemaps
+
+Site map genreation for Vanilla Forums, using the [site map protocol](https://www.sitemaps.org/protocol.html).
+
+## Site Map Index
+
+Each community generates a /sitemapindex.xml page, containing a site map for each category. There is a maximum of 50,000 site maps.
+
+## Site map
+
+Each category has its own sitemap, containing links to each discussion. Again, the limit is 50,000 urls.
+
+## Filtering results
+
+If you need to filter out the results, there is an event (siteMapCategories) after the categories are fetched to filter out the results.
+
+---
+Copyright &copy; 2017 [Vanilla Forums](http://vanillaforums.com).
```

Comment:
Better to use the same format everywhere. Easier to update on a new year that way!